change/@fluentui-react-9de77fe1-92c7-490f-8000-04c3eb7aa8e8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30369&commitId=98f39d524a2563fb20d98a6d9b483ea285f0d0bd&env=prod&clientType=FLUENTUI)


  <details><summary><b>Chat</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Chat.Chat Message Example Styled.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>